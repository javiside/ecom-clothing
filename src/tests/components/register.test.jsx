import React from "react";
import { shallow } from "enzyme";

import Register from "../../components/register";

describe("Register", () => {
  it("should mount the component correctly", () => {
    expect(shallow(<Register />)).toMatchSnapshot();
  });
});
