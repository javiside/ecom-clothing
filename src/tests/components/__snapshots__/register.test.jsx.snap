// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register should mount the component correctly 1`] = `
<div
  className="register"
>
  <h2
    className="heading"
  >
    I do not have an account
  </h2>
  <span
    className="title"
  >
    Register with your name, email and password
  </span>
  <span
    className="register-error"
  />
  <form
    className="register-form"
    onSubmit={[Function]}
  >
    <FormInput
      label="Display Name"
      name="displayName"
      onChange={[Function]}
      required={true}
      type="text"
      value=""
    />
    <FormInput
      label="Email"
      name="email"
      onChange={[Function]}
      required={true}
      type="email"
      value=""
    />
    <FormInput
      autoComplete="on"
      label="Password"
      name="password"
      onChange={[Function]}
      required={true}
      type="password"
      value=""
    />
    <FormInput
      autoComplete="on"
      label="Confirm Password"
      name="confirmPassword"
      onChange={[Function]}
      required={true}
      type="password"
      value=""
    />
    <CustomButton
      type="submit"
    >
      REGISTER
    </CustomButton>
  </form>
</div>
`;
